_schemaV2.graphql

```diff
@@ -11279,6 +11333,53 @@ type HomePageSalesModule {
   results: [Sale]!
 }
 
+# Experimental schema for new home view
+type HomeView {
+  sectionsConnection(
+    after: String
+    before: String
+    first: Int
+    last: Int
+  ): HomeViewSectionConnection
+}
+
+# A component specification
+type HomeViewComponent {
+  # Which component type to render
+  type: HomeViewComponentType
```

Comment:
(late here sorry 🙇 ): Is there any reason we're following a pattern different than GraphQL __typename union here?
My question is more about creating a different deviation for a type's `type` rather than using the standards we have.

Reply:
cc @MounirDhahri